db/seeds.rb

```diff
@@ -86,6 +86,17 @@
     r.start = Tod::TimeOfDay("16:00")
     r.stop = Tod::TimeOfDay("23:00")
   end.save!
+
+  Widget.find_or_initialize_by(
+    name: "Buses",
+    location: location
+  ) do |r|
+    r.enabled = true
+    r.duration_seconds = 20
+    r.position = 5
+    r.sidebar_text = 'Bus Routes and Positions'
+    r.show_weather = true
+  end.save!
```

Comment:
How are you adding buses to the database? Did you miss adding a migration and the model? It doesn't look like the frontend is connected to the rails graphql API for this change. I think it would make sense to split out the frontend and back end changes into separate tickets.

Reply:
I'm not adding them to the database yet - and I should've made that clear in the PR description. Right now there's a json file taken from an arbitrary time from the above RIPTA transit API site, I was using that formatting to create pins. As it stands, the map cannot update itself.